doc/reference/guidelines.rst

```diff
@@ -4,30 +4,37 @@
 Odoo Guidelines
 =================
 
-This page introduce the new Odoo Coding Guidelines. This guidelines
-aims to improve the quality of the code (better readability of source,
-...) but also to improve Odoo Apps ! Indeed, a proper code will ease
+This page introduces the new Odoo Coding Guidelines. These guidelines
+aim to improve the quality of the code (better readability of source...)
+but also to improve Odoo Apps! Indeed, a proper code will ease
 the maintenance and debugging, lower the complexity and promote the
 reliability.
 
+These guidelines should be applied to every new module. These guidelines will be applied to old module only in case of refactoring.
+
 Module structure
 ================
 
 Directories
 ------------
-A module is organised in a few directory :
+A module is organised in some important directories. These directories aim to contain the business core of the module; having a look at them should make understand the purpose of the module.
```

Comment:
I suggest having subdirectories to be optional for modules with few file. The obvious case would be the module extending one model and one view. In those cases we would instead end the file names with append `_model.py`or _view.xml`.